applications/fishing-map/src/features/timebar/Timebar.tsx

```diff
@@ -65,11 +66,16 @@ const TimebarWrapper = () => {
   const { bounds } = useMapBounds()
   const { sourcesFeatures, haveSourcesLoaded, sourcesMetadata } = useActivityTemporalgridFeatures()
   const debouncedBounds = useDebounce(bounds, 400)
+  const windowWidth = window?.innerWidth
+  const [isMobile, setIsMobile] = useState<boolean>(
```

Comment:
This would be nice to be available for other components as a hook but it is not needed for now